### PR TITLE
Remove sys_policy from special handling as it's implemented in

### PR DESCRIPTION
**Pull request by jefferai**
logical_system too. Clean up the mux handlers.

### CODE REVIEW

File: http/handler.go [hunk 1]
```diff
@@ -24,28 +24,14 @@ func Handler(core *vault.Core) http.Handler {
...
 	mux.Handle("/v1/sys/leader", handleSysLeader(core))
 	mux.Handle("/v1/sys/health", handleSysHealth(core))
-	mux.Handle("/v1/sys/rotate", proxySysRequest(core))
-	mux.Handle("/v1/sys/key-status", proxySysRequest(core))
 	mux.Handle("/v1/sys/generate-root/attempt", handleSysGenerateRootAttempt(core))
 	mux.Handle("/v1/sys/generate-root/update", handleSysGenerateRootUpdate(core))
 	mux.Handle("/v1/sys/rekey/init", handleSysRekeyInit(core))
-	mux.Handle("/v1/sys/rekey/backup", proxySysRequest(core))
 	mux.Handle("/v1/sys/rekey/update", handleSysRekeyUpdate(core))
+	mux.Handle("/v1/sys/", proxySysRequest(core))
```
Comment (vishalnayak) [inline http/handler.go]: I wonder if we really need `proxySysRequest()`. It is just `handleLogical(core, true)`.

**Comment by vishalnayak** [comment]
Yes. List policy response was the only distinguishing factor.
LGTM! :ship:

File: http/handler.go (hunk 1 above)
Comment (jefferai) [inline http/handler.go]: That is a totally fair point.

Comment (jefferai) [inline http/handler.go]: I'll merge and then fix that up.

**Pull request merged**